splink/linker.py

```diff
@@ -1002,110 +1002,6 @@ def _raise_error_if_necessary_accuracy_columns_not_computed(self):
                 "Please re-run your linkage with it set to True."
             )
 
-    def load_settings(
-        self,
-        settings_dict: dict | str | Path,
-        validate_settings: str = True,
-    ):
-        """Initialise settings for the linker.  To be used if settings were
```

Comment:
I think we can completely eliminate this now we're forcing settings to be passed on creation